client/components/thank-you/index.tsx

```diff
@@ -115,36 +104,63 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 	const translate = useTranslate();
 
 	const {
+		headerClassName,
 		containerClassName,
 		sections,
 		showSupportSection = true,
 		thankYouTitle,
+		thankYouSubtitle,
 		thankYouImage,
+		headerTextColor,
+		headerBackgroundColor,
```

Comment:
Also, to expand on my comment RE defaulting them in, it may be better to do that here.
```suggestion
		headerTextColor = 'white',
		headerBackgroundColor = '#0675C4',
```